### DISCUSSION TITLE
"Erase all" button for calendar

### DISCUSSION THREAD

**Discussion by MairaBelm** [gaborauth/toolsboox-android#97] [Ideas]
While testing the new Marks calendar feature, I noticed that the Eraser tool works inconsistently, leaving invisible traces or even adding its own traces sometimes. As a result, marks are not always removed after erasing all visible scribbling. Pressing "Update marks" does not make any difference. 
I am hesitant to open an issue for this because this is probably a Onyx firmware glitch with less effect on other apps that are not relying on scribbling status, but I think an "Erase all scribbles" button or a "Clear marks" one would be a nice addition in case the Eraser keeps misbehaving.

**Comment by gaborauth** [comment]
So, it's done, will be in the 1.5.9-xx release. :)

**Reply by MairaBelm** [reply]
Great! I have just updated the Spanish translation and a question arose: will this button erase just strokes or calendar entries as well? Might be worth specifying this in the confirmation prompt.

**Reply by gaborauth** [reply]
It deletes the actually displayed page content, regardless it is calendar page or notes.

**Reply by MairaBelm** [reply]
Sorry, I meant "imported calendar events". I just tested this functionality and saw it just deleted handwritten strokes. Because the message says "the whole content", I think this might be confusing for first-time users, though.

**Reply by gaborauth** [reply]
Oh, I see. Hm... they cannot erase the printed text, so, I think it will be good enough, and in a week or two I will ask the users what they think about it. :)
